core/controllers/editor_test.py

```diff
@@ -2435,7 +2435,7 @@ def setUp(self):
         self.answer = 'This is an answer'
         self.answer_details = 'These are the answer details'
         self.state_reference = (
-            stats_models.LearnerAnswerDetailsModel
+            stats_services
             .get_state_reference_for_exploration(self.exp_id, self.state_name))
```

Comment:
Does this function exist in `stats_services` already?

Reply:
Yeah it exists